### PR TITLE
Agent data on commandlog

### PR DESCRIPTION
**Pull request by mmadariaga**
Set user data on Commandlog. Includes:
 - Ip
 - Admin id

### CODE REVIEW

File: web/admin/application/library/IvozProvider/Klear/Auth/User.php [hunk 1]
```diff
@@ -150,4 +150,9 @@ public function getUniqueIdenForCache()
         // Devuelve un hash basado en las propiedades publicas
         return md5(print_r(call_user_func('get_object_vars', $this), true));
     }
+
+    public function __toString()
+    {
+        return 'Administrator#' . $this->id;
```
Comment (Kaian) [inline web/admin/application/library/IvozProvider/Klear/Auth/User.php]: I can't see `id` between this class attributes. :thinking: 
Maybe `_id` from `Klear_Model_User`?

File: web/admin/application/library/IvozProvider/Klear/Dynamic/Config/Base.php [hunk 2]
```diff
@@ -69,8 +70,10 @@ public function init($siteConfig)
     {
         $this->_siteConfig = $siteConfig;
         if (\Zend_Auth::getInstance()) {
+            /** @var \IvozProvider\Klear\Auth\User $identity */
             $identity = \Zend_Auth::getInstance()->getIdentity();
             if ($identity) {
+                DataGateway::$user = $identity->__toString();
```
Comment (Kaian) [inline web/admin/application/library/IvozProvider/Klear/Dynamic/Config/Base.php]: what about `DataGateway::$user = (string) $identity`;

File: library/Ivoz/Provider/Domain/Model/Commandlog/CommandlogAbstract.php [hunk 3]
```diff
@@ -307,6 +316,33 @@ public function getArguments()
...
+     * Set agent
+     *
+     * @param array $agent
+     *
+     * @return self
+     */
+    protected function setAgent($agent = null)
+    {
+        if (!is_null($agent)) {
+        }
```
[the remark this answers is not in the document]
Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Model/Commandlog/CommandlogAbstract.php]: This is a generator error. I'll fix it into another PR

File: web/admin/application/library/IvozProvider/Klear/Auth/User.php (hunk 1 above)
Comment (mmadariaga) [reply web/admin/application/library/IvozProvider/Klear/Auth/User.php]: Is attribute is injected in Klear\Auth\Adapter, so they coexist for some reason :(

I'll use _id anyway, it's a better option

**Pull request merged**